packages/suite/src/views/onboarding/steps/SecurityCheck/SecurityCheck.tsx

```diff
@@ -119,6 +124,9 @@ const firmwareInstalledChecklist = [
     },
 ] as const;
 
+const isAuthenticityCheckSupported = (deviceModel: DeviceModelInternal) =>
+    [DeviceModelInternal.T2B1, DeviceModelInternal.T3T1].includes(deviceModel);
```

Comment:
Here I would add a map `Record<DeviceModelInternal, boolean>` so when adding a new device we have to explicitly define if it has AUthenticity check.

Reply:
9822eb26d306fd5f3bc4b83bf3b280221025dd70